### PR TITLE
Typos from Perl5 (2011 - Episode 3)

### PR DESCRIPTION
**Pull request by Fourchaux**
Some typos from https://perl5.git.perl.org/perl.git/search/HEAD?s=typo;st=free

### CODE REVIEW

**Comment by larsoner** [comment]
@Fourchaux can you rebase? There is a merge conflict

**Comment by Fourchaux** [comment]
The merge conflict is still there?

**Comment by Fourchaux** [comment]
Is this PR problematic?
(And if so, what is the problem?)

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3457,6 +3476,7 @@ contols->controls
 contorls->controls
 contoroller->controller
 contradically->contradictory
+contrained->contained
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: also constrained

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9951,6 +9999,7 @@ opthalmology->ophthalmology
 opthamologist->ophthalmologist
 optiional->optional
 optimasation->optimization
+optimation->optimisation
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: should probably add (and prefer) `optimization`

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Fourchaux) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Fourchaux) [reply codespell_lib/data/dictionary.txt]: Done

**Comment by arm-in** [comment]
Pardon me for being obtrusive, but the three removals are not obvious to me. Why this?

**Comment by larsoner** [comment]
I did not notice those, I agree they look like errant removals

**Comment by Fourchaux** [comment]
Re-adding these 3 removals
with https://github.com/codespell-project/codespell/pull/1053

**Pull request merged**